### DISCUSSION TITLE
Box dissapears when I click on ARButton

### DISCUSSION THREAD

**Discussion by cjoejoe1** [pmndrs/xr#197] [Q&A] [answered]
Hi, I have a simple boxGeometry in the middle of my screen with a ARButton from react-xr.

When I click on the ARButton, AR camera opens but the Box disappears. Same on desktop and on mobile.

I feel like I've missed something really stupid.. :-/  

Thanksss for your help,

`import { ARButton, XR } from '@react-three/xr'
import { Canvas, useFrame } from '@react-three/fiber'
import {OrbitControls} from '@react-three/drei'

function Box(props) {

  return (
    <mesh
    {...props}
    >
    <boxGeometry args={[1, 1, 1]} />
    <meshStandardMaterial color={'red'} />
    <OrbitControls/>
  </mesh>
  )
}

export default function BoxPage() {
  return (
    <>
      <ARButton/>
      <Canvas>
        <XR>
          <ambientLight />
          <pointLight position={[20, 10, 10]} />
          <Box position={[0, 0, 0]} />
        </XR>
      </Canvas>
    </>
  )
}
`

**Comment by CodyJasonBennett** [comment, marked as answer]
The viewer camera is usually positioned at 0, 0 although at an adjusted height. Maybe try double sided material or move the box forward/back on the z axis.

**Reply by cjoejoe1** [reply]
Well that was easy :-) Brilliant thanks!!
